src/gui/accelerators.c

```diff
@@ -534,14 +534,15 @@ void dt_accel_connect_lua(const gchar *path, GClosure *closure)
   gtk_accel_group_connect_by_path(darktable.control->accelerators, accel_path, closure);
 }
 
-void dt_accel_connect_manual(GSList *list, const gchar *full_path, GClosure *closure)
+void dt_accel_connect_manual(GSList **list_ptr, const gchar *full_path, GClosure *closure)
 {
   gchar accel_path[256];
   dt_accel_path_manual(accel_path, sizeof(accel_path), full_path);
   dt_accel_t *accel = _lookup_accel(accel_path);
   accel->closure = closure;
   gtk_accel_group_connect_by_path(darktable.control->accelerators, accel_path, closure);
-  list = g_slist_prepend(list, accel);
```

Comment:
The result is assigned to a local variable, which is never read afterwards.

Reply:
but now this function doesn't prepend the accel to the list any more